tests/cxx/namespace_alias-d1.h

```diff
@@ -0,0 +1,20 @@
+//===--- namespace_alias-d1.h - test input file for IWYU ------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef INCLUDE_WHAT_YOU_USE_TESTS_CXX_NAMESPACE_ALIAS_D1_H_
+#define INCLUDE_WHAT_YOU_USE_TESTS_CXX_NAMESPACE_ALIAS_D1_H_
+
+// Just declare something in a namespace. Nested for good measure.
+namespace i1_ns1 {
```

Comment:
I actually forgot to ask for this for the prior PR, but I'm trying to move tests away from the filename-prefixed naming of symbols. I wrote up some test guidelines for this a while back: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUStyle.md#symbol-names

Could you rename the namespaces to reflect their role, e.g. `ns_top`, `ns_nested`, `ns_alias` or something?

I think that will both make the test clearer and make it easier to rename symbols/files.

Reply:
Or maybe even just `ns1` and `ns2` for the nested namespaces, actually.